### PR TITLE
Bug 345

### PR DESCRIPTION
**Pull request by aronmi**
adds a bool prop to TR and TD components which MessageRow passes true, or anyone who wants to render a row or cell just for styling ( not containing data ), TR & RD onClick() checks the prop

### CODE REVIEW

File: packages/gridjs/src/view/table/messageRow.tsx [hunk 1]
```diff
@@ -15,10 +15,11 @@ export interface MessageRowProps extends BaseProps {
 export class MessageRow extends BaseComponent<MessageRowProps, {}> {
   render() {
     return (
-      <TR>
+      <TR stylingRow={true}>
```
Comment (afshinm) [inline packages/gridjs/src/view/table/messageRow.tsx:18]: I like this approach but I don't think `stylingRow` is the best attribute name that can be used here. Can we change it to something a bit more self-describing?

File: packages/gridjs/src/view/table/td.tsx [hunk 2]
```diff
@@ -10,11 +10,12 @@ import { PluginRenderer } from '../../plugin';
 
 export interface TDProps
   extends BaseProps,
-    JSX.HTMLAttributes<HTMLTableCellElement> {
```
Comment (afshinm) [inline packages/gridjs/src/view/table/td.tsx:13]: can you run prettifier on this please?

File: packages/gridjs/tests/view/table/message-row.test.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { mount } from 'enzyme';
+import { createContext, h } from 'preact';
+import { Config } from '../../../src/config';
+import { EventEmitter } from '../../../src/util/eventEmitter';
+import { TableEvents } from '../../../src/view/table/events';
+import { MessageRow } from '../../../src/view/table/messageRow';
+
+describe('MessageRow component', () => {
```
Comment (afshinm) [inline packages/gridjs/tests/view/table/message-row.test.tsx:8]: I really like these tests! :+1:

File: packages/gridjs/src/view/table/messageRow.tsx (hunk 1 above)
Comment (aronmi) [reply packages/gridjs/src/view/table/messageRow.tsx:18]: i think so. suggest please

Comment (afshinm) [reply packages/gridjs/src/view/table/messageRow.tsx:18]: `messageRow`?

File: packages/gridjs/src/view/table/td.tsx (hunk 2 above)
Comment (aronmi) [reply packages/gridjs/src/view/table/td.tsx:13]: this happens after prettifier, i double cheched in prettifier.net, same result.

**Comment by afshinm** [comment]
thanks @aronmi :tada:

**Pull request merged**